packages/web3-eth-accounts/src/wallet.ts

```diff
@@ -95,6 +199,16 @@ export class Wallet<
 		return Object.values(this._accounts)[addressOrIndex];
 	}
 
+	/**
+	 * Removes an account from the wallet.
+	 * @param addressOrIndex - The account address, or index in the wallet.
+	 * @returns true if the wallet was removed. false if it couldn’t be found.
+	 * ```ts
+	 * web3.eth.accounts.wallet.add('0xbce9b59981303e76c4878b1a6d7b088ec6b9dd5c966b7d5f54d7a749ff683387');
+	 *
+	 * web3.eth.accounts.wallet.remove('0x85D70633b90e03e0276B98880286D0D055685ed7'); // FIX THIS
```

Comment:
```suggestion
	 * web3.eth.accounts.wallet.remove('0x85D70633b90e03e0276B98880286D0D055685ed7'); 
```
Should this be removed? or is there any thing needs to be fixed?

Reply:
This is fixed in nazars [pr](https://github.com/ChainSafe/web3.js/pull/5175#event-6900514583) that was merged. will update now